app/src/main/java/org/oppia/app/viewmodel/ObservableViewModel.kt

```diff
@@ -0,0 +1,31 @@
+package org.oppia.app.viewmodel
```

Comment:
Note to others: we probably should introduce this in a separate PR. Please do this directly; don't wait for me since I'm focused on infrastructure work right now.

@rt4914